cli/cmd/policy.go

```diff
@@ -183,6 +184,12 @@ func inputPolicy(cmd *cobra.Command) (string, error) {
 	if policyCmdState.URL != "" {
 		return inputPolicyFromURL(policyCmdState.URL)
 	}
+	// if running via stdin
+	stat, _ := os.Stdin.Stat()
+	if (stat.Mode() & os.ModeCharDevice) == 0 {
```

Comment:
I can see what we are trying to do here, support both "pipes" and "code injection" (via <)

I think I like the pipe idea, but the "code injection" not so much since we already have a flag for passing a file from disk. 

Could we limit this PR to enable only pipes using `os.ModeNamedPipe` rather than `os.ModeCharDevice`?

Reply:
What I have used in the past is `if (stat.Mode() & os.ModeNamedPipe) != 0 {}` but if it works with your implementation Im ok with it.